### PR TITLE
fixed issues related to account and onboarding page

### PR DESCRIPTION
**Pull request by ascode94**

### CODE REVIEW

File: src/components/views/onboarding/Onboarding.view.tsx [hunk 1]
```diff
@@ -48,7 +48,9 @@ enum LabelStatus {
 const OnboardHeader: FC<IOnboard> = ({ step }) => {
 	return (
 		<OnboardHeaderConatiner>
-			<H5 weight={700}>Complete your profile</H5>
+			<H5 style={{ textAlign: 'center' }} weight={700}>
+				Complete your profile
+			</H5>
```
Comment (MohammadPCh) [inline src/components/views/onboarding/Onboarding.view.tsx:53]: use styled-components for styling instead of adding inline styles.

**Review by MohammadPCh — CHANGES_REQUESTED** [review]
Thank you @ascode94,
Please fix the tiny change request.

File: src/components/views/onboarding/Onboarding.view.tsx (hunk 1 above)
Comment (ascode94) [reply src/components/views/onboarding/Onboarding.view.tsx]: Thanks a lot for the review @MohammadPCh  🤩
I fixed it.

**Review by MohammadPCh — APPROVED** [review]
Thank you @ascode94

**Pull request merged**